WebApp/static/core/js/views/desktop/Main.js

```diff
@@ -2473,3 +2473,14 @@ Ext.define('Ext.dirac.views.desktop.Main', {
     /*-----------------END - IMPLEMENTATION OF THE INTERFACE BETWEEN STATE MANAGEMENT ADN DESKTOP----------------*/
 
   });
+  /*
+   * Welcome window
+   */
+  if (GLOBAL.APP.configData.user.username) {
```

Comment:
This is not following the ExtJS developer convention. I know it is JS and this code can be executed everywhere, but you must add this code inside a class. What is a class in ExtJs:

Ext.define('Ext.dirac.views.desktop.Main', {
  extend : 'Ext.panel.Panel',
.....
});
In addition I do not like it at all. Every time when I open or refresh my browser I get this message. I have to click to close. Most of the users know what is DIRAC.  You also added a hyper link, so if someone click on it the actual web page will be overwritten.
Conclusion:
-In this format can not be approved. You should add this few lines inside a class.
-I do not like this features, I would say we must discuss this in the BILD meeting.